Gopher-Industries/Team-TaskforceX
# 211-Fonts-Colours-layout-Scroll

Summary of changes:

-Added scroll function.
-Changed fonts and weights.
-Added nutritional summary with hairlines.
-Changed margins to bring the screen in-line with wireframe. 
-Changed colours to match wireframe, including background. 
-Added additional comments.
-Cleaned up code.
-Added and confirmed function of continue button flow to "Meal Planning" 
-Added and confirmed function of back arrow flow to "Your Preferences. (Need to change arrow to correct arrow)


For future students: 

 -Need to take nutritional summary that will be generated by all the user inputs and information during onboarding and dynamically display this as their nutritional plan.
-Need to take user inputs from "Health Conditions" and feed them into this screen under "Health Conditions".

Testing Instructions:

-Confirm continue button flows to "Meal Planning". 
-Confirm back arrow returns to "Your Preferences"

![Plan 1](https://user-images.githubusercontent.com/118961659/207213736-13020cbe-09e3-4b4b-ada3-2c9441d2bb4c.PNG)
![Plan 2](https://user-images.githubusercontent.com/118961659/207213741-c0ad2cb7-2c31-4e38-92cf-02ce5f9950f6.PNG)
